extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -90,19 +91,25 @@ describe('integration test: a4a', () => {
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
     xhrMock = sandbox.stub(Xhr.prototype, 'fetch');
+    // Expect key set fetches for signing services.
+    const fetchJsonMock = sandbox.stub(Xhr.prototype, 'fetchJson');
+    for (const serviceName in signingServerURLs) {
```

Comment:
Yeah, I've been torn on this.  Given the slight performance advantage and that the style guide does allow so long as underlying object is not an array, I'm going to leave as is.